include/mo.php

```diff
@@ -78,14 +78,30 @@ public function import_from_db( $lang ) {
 	}
 
 	/**
-	 * Returns the post id of the post storing the strings translations.
+	 * Returns the post ID of the post storing the strings translations.
 	 *
 	 * @since 1.4
 	 *
 	 * @param PLL_Language $lang The language object.
-	 * @return int
+	 * @return int|null
+	 *
+	 * @phpstan-return positive-int|null
 	 */
 	public static function get_id( $lang ) {
+		return self::get_id_from_term_id( $lang->get_tax_prop( 'language', 'term_id' ) );
```

Comment:
As finally we keep it.
```suggestion
		return self::get_id_from_term_id( $lang->term_id );
```